Library/Formula/nginx.rb

```diff
@@ -121,12 +120,7 @@ def post_install
     end
   end
 
-  test do
-    system "#{bin}/nginx", "-t"
-  end
-
   def passenger_caveats; <<-EOS.undent
-
```

Comment:
(Not actually your fault on this. Our audit does flag it, and we should probably look at an alternative way to avoid them rubbing against each other).